src/main/webapp/index.html

```diff
@@ -8,13 +8,38 @@
   </head>
   <body>
     <header>
-      <h1>smallcity</h1>
+      <h1 class="logo">smallcity</h1>
+      <div class="zipcode-container">
+        your zipcode: 
+      </div>
     </header>
 
     <div class="main-container">
       <div class="map-container">
       </div>
       <div class="results-container">
+        <div class="results-card">
+          <div class="results-image">
+            image
+          </div>
+          <div class="results-business-description">
+            <h2>Business Name</h2>
+            <p>Business address</p>
+            <p>Vicinity</p>
+          </div>
+          <div class="results-rating"></div>
+          <button class="results-website-button">Visit Website</button>
+        </div>
+        <div class="results-card">
+          
+        </div>
+        <div class="results-card">
+          
+        </div>
+        <div class="pagination-container">
+          <button class="pagination-buttons">Back</button>
```

Comment:
nit: Previous and next might make a little bit more sense then forward and back

Reply:
I agree, changed